tiff-2-png-converter.py

```diff
@@ -7,100 +7,249 @@
 import math
 import tifffile as tiff
 import numpngw
-from progress_bar import print_progress_bar
 import image_process_helper as iph
 from scipy.interpolate import interp1d
+import logging
 import re
 
 import typer
 
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s [%(levelname)s] %(message)s",
+    handlers=[logging.FileHandler("tiff-2-png.log"), logging.StreamHandler()],
+)
+
+app = typer.Typer()
+
+arg_values = {
+    "verbose": False,
+    "equalize_histogram": False,
+    "equalize_histogram_method": None,
+    "gamma": False,
+    "gamma_val": 1,
+    "nimgs": None,
+    "source": "",
+    "dest": "",
+}
+
+
+def list_folder_files(path_to_foder):
+
```

Comment:
Add [docstring](https://www.python.org/dev/peps/pep-0257/) for function description